docs/references.md

```diff
@@ -7,6 +7,7 @@ Links to special software packages are located at the end of the according softw
 
 - Twitter: <https://twitter.com/DietPi_>
 - Facebook: <https://www.facebook.com/groups/167403007229675>
+- DietPi Blog page: <https://dietpi.com/blog>
```

Comment:
What do you think of naming it just "Blog", to have a single word list "Twitter" and "Facebook"?

Reply:
Ok.